RepoInspector/RepoInspector/Analysis/Scoring/CodeFenceScorer.cs

```diff
@@ -30,6 +33,7 @@ public override int Count(PullRequest prDetails)
         public override IEnumerable<string> Extract(string s) =>
             string.IsNullOrWhiteSpace(s)
                 ? Enumerable.Empty<string>()
-                : _codeFence.Matches(s).Select(m => m.Value);
+                // Only select *closed* fenced blocks
+                : Markdown.Parse(s, _markdownPipeline).OfType<FencedCodeBlock>().Where(fcb => !fcb.IsOpen).Select(fcb => fcb.Lines.ToString());
```

Comment:
What about indented (rather than fenced) code blocks? Does this pick those up, too?

Reply:
I believe I tested, and it did, but let me check...the "fence" is interpreted somewhat loosely.